### DISCUSSION TITLE
'auto-editor' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by 3xploiton3** [WyattBlue/auto-editor#394] [1. Help] [answered]
OS: Windows 11 22H2
Python 3.12.0

pip list
Package     Version
ae-ffmpeg   1.1.2
auto-editor 23.43.1
numpy       1.26.1
Pillow      10.1.0
pip         23.3.1
pyav        11.2.0

![image](https://github.com/WyattBlue/auto-editor/assets/19517680/528d01bc-37e6-4abd-aa86-c3b54ada420b)

**Comment by 3xploiton3** [comment, marked as answer]
problem solved.

add to environment variables > User Variables > Path > Edit > New

` C:\Users\<Your Username>\AppData\Roaming\Python\Python312\Scripts`

**Comment by Matasuegra466** [comment]
I dont know why but the program dont work any more in my pc 
I try installing the program in other pc and it still not working 
I try upgrading the program and still not working 
Seems like the program dont cant cut the video y try a lot of things but i can fix it
